src/algorithms/QPAnsatz.m

```diff
@@ -0,0 +1,93 @@
+classdef QPAnsatz
+    % Quasi-Particle excitation ansatz
+    
+    properties
+        alg_eigs = Arnoldi('MaxIter', 100, 'KrylovDim', 30, 'Tol', 1e-8, 'verbosity', Verbosity.diagnostics)
```

Comment:
keyword arguments with capitals: `Verbosity, Verbosity.diagnostics` (matlab ignores capitalization, so this is just a style thing